content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -147,22 +145,22 @@ The payload may already be stored in another storage/package manager, which can
 python ${SCRIPTS}create_signed_statement.py \
```

Comment:
its is usually python3.
on my laptop the cmd 'python' does not exist

Reply:
ah that is what I had originaly, but I think someone changed it to python3. happy to change it back

Reply:
python3 -m used consistently now (I hope)